### PR TITLE
Feature is vpesg integration

### PR DESCRIPTION
**Pull request by SunithaGudisagarIBM**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TEST=./ibm TESTARGS='-run=TestAccIBMISVirtualEndpointGateway_Basic_SecurityGroups'

```
Screenshot 2022-01-11 at 1 39 10 PM

### CODE REVIEW

File: website/docs/r/is_virtual_endpoint_gateway.html.markdown [hunk 1]
```diff
@@ -88,7 +92,8 @@ Review the argument references that you can specify for your resource.
   **NOTE**: `id` and `subnet` are mutually exclusive.
 
 - `resource_group` - (Optional, Forces new resource, String) The resource group ID.
-- `tags`- (Optional, Array of Strings) A list of tags associated with the instance.
+- `security_groups` - (Optional, list) The security groups to use for this endpoint gateway. If unspecified, the VPC's default security group is used.
+  **NOTE:** either of `ibm_is_security_group_target` resource or `security_groups` attribute should be used, both can't be use together. - `tags`- (Optional, Array of Strings) A list of tags associated with the instance.
```
Comment (sumitkumartiwari) [inline website/docs/r/is_virtual_endpoint_gateway.html.markdown:96]: Put the `- tag `on next line

File: website/docs/d/is_virtual_endpoint_gateway.html.markdown [hunk 2]
```diff
@@ -54,4 +54,6 @@ In addition to the argument reference list, you can access the following attribu
   - `name` - (String) The target name.
   - `resource_type` - (String) The resource type of the subnet reserved IP.
 - `vpc` - (String) The VPC ID.
+- `security_groups` - The security groups to use for this endpoint gateway.
```
Comment (hkantare) [inline website/docs/d/is_virtual_endpoint_gateway.html.markdown:57]: Can you add the type of the security_groups - (List)

File: website/docs/d/is_virtual_endpoint_gateways.html.markdown [hunk 3]
```diff
@@ -53,3 +53,5 @@ In addition to the argument reference list, you can access the following attribu
     - `name` - (String) The endpoint gateway target name.
     - `resource_type` - (String) The endpoint gateway target resource type.
   - `vpc` - (String) The VPC ID.
+  - `security_groups` - The security groups to use for this endpoint gateway.
```
Comment (hkantare) [inline website/docs/d/is_virtual_endpoint_gateways.html.markdown:56]: Same as above

**Comment by hkantare** [comment]
Can we run the  all the  testcase releted to endpoint gateway and share the results

**Comment by ujjwal-ibm** [comment]
vpesg
vpesg1

**Comment by SunithaGudisagarIBM** [comment]
Screenshot 2022-01-27 at 10 15 43 AM

**Pull request merged**